src/coreclr/tools/aot/ILCompiler/ILCompiler.csproj

```diff
@@ -33,7 +33,10 @@
   <PropertyGroup>
     <ILCompilerVersion>6.0.0-preview.5.21222.2</ILCompilerVersion>
 
-    <ObjWriterArtifactPath Condition="'$(TargetsWindows)' == 'true'">$(ArtifactsDir)llvm-project\llvm\build\$(TargetArchitecture)\Release\bin\$(LibPrefix)objwriter$(LibSuffix)</ObjWriterArtifactPath>
+    <_ObjWriterBuildType>$(Configuration)</_ObjWriterBuildType>
```

Comment:
I usually build by running `./build.sh nativeaot -rc Debug -lc Release`. If I run `./build.sh nativeaot.objwriter -rc Debug -lc Release`, does `$(Configuration)` map to `rc` parameter? That would be my expectations in that case.

Reply:
Yup. I use the same workflow.